guides/common/modules/proc_customizing-the-discovery-pxe-boot.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Customizing_the_Discovery_PXE_Boot_{context}"]
+= Customizing the Discovery PXE boot
+[[Discovery_Templates_and_Snippets_Settings_{context}]]
+[[Provisioning_Template_PXELinux_Discovery_Snippet_{context}]]
+
+{Project} builds PXE boot menus from the global provisioning templates: `PXELinux global default` for BIOS provisioning, `PXEGrub global default`, and `PXEGrub2 global default` for UEFI provisioning.
+The PXE boot menus are available on {ProjectServer} and all {SmartProxies} that have the TFTP feature enabled.
+
+The Discovery menu item uses a Linux kernel for the operating system, which passes kernel parameters to configure the Discovery service.
+You can customize the passed kernel parameters by changing the following snippets:
+
+Snippet `pxelinux_discovery`:: This snippet is included in the `PXELinux global default` template.
++
+This snippet renders the Discovery boot menu option.
+The `KERNEL` and `APPEND` options boot the Discovery kernel and initial RAM disk.
+The `APPEND` option contains kernel parameters.
+
+Snippet `pxegrub_discovery`:: This snippet is included in the `PXEGrub global default` template.
+However, Discovery is http://projects.theforeman.org/issues/15997[not supported with GRUB 1.x].
+
+Snippet `pxegrub2_discovery`:: This snippet is included in the `PXEGrub2 global default` template.
++
+This snippet renders the Discovery GRUB2 menu entry.
+The `common` variable contains kernel parameters.
+
+To view provisioning templates and snippets, navigate to *Hosts* > *Templates* > *Provisioning Templates*.
+Templates and snippets are locked to prevent changes.
+If you want to edit a template or snippet, clone it, save it with a unique name, and then edit the clone.
+
+For information about the kernel parameters, see xref:kernel-parameters-for-discovery-customization_{context}[].
+
+.Procedure
+. Clone and edit the snippet you want to customize.
+For more information, see xref:cloning-provisioning-templates_{context}[].
+. Clone and edit the template that contains the original snippet.
+Include your custom snippet instead of the original snippet.
+For more information, see xref:cloning-provisioning-templates_{context}[].
+. Set your custom template in global settings.
+In the {ProjectWebUI}, navigate to *Administer* > *Settings* and click the *Provisioning* tab and set your template as a `Global default PXE* template`.
+. Build the global default PXE boot menus.
```

Comment:
```suggestion
. Navigate to  *Hosts* > *Templates* > *Provisioning Templates* and click *Build PXE Default*.
```
You do not need to mention the UI multiple times if the procedure is in the same UI. You only need to mention the UI if the user has to jump between UIs or between the UI and CLI.